dashboard/app/views/pd/workshop_mailer/teacher_enrollment_reminder.html.haml

```diff
@@ -18,6 +18,19 @@
     = @workshop.subject
     workshop.
 
+- if @workshop.pre_survey?
+  - survey_link = link_to 'survey', @pre_survey_path
+  - if @days == 10
```

Comment:
This seems magic-number-y. I suggest defining 10 as a constant in workshop_mailer, and renaming days to days_before